ops-bedrock/docker-compose.yml

```diff
@@ -98,8 +98,6 @@ services:
       OP_PROPOSER_ROLLUP_RPC: http://op-node:8545
       OP_PROPOSER_POLL_INTERVAL: 1s
       OP_PROPOSER_NUM_CONFIRMATIONS: 1
-      OP_PROPOSER_SAFE_ABORT_NONCE_TOO_LOW_COUNT: 3
-      OP_PROPOSER_RESUBMISSION_TIMEOUT: 30s
```

Comment:
These flags are part of the `txmgr`'s cli flags [here](https://github.com/ethereum-optimism/optimism/blob/f9928f1e3885bd556171f541046001a6278537d9/op-service/txmgr/cli.go#L76-L87) I believe. Should be ok to delete but just wanted to point out I think they are used, just optional.

Reply:
yea. All of these flags should be defaulted.